source/exe/BUILD

```diff
@@ -134,25 +142,6 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
-    name = "envoy_common_with_core_extensions_lib",
-    deps = [
-               "//source/common/event:libevent_lib",
-               "//source/common/network:utility_lib",
-               "//source/common/stats:stats_lib",
-               "//source/common/stats:thread_local_store_lib",
-               "//source/server:drain_manager_lib",
-               "//source/server:listener_hooks_lib",
-               "//source/server:options_lib",
-               "//source/server:server_base_lib",
-           ] + envoy_all_core_extensions() +
-           # TODO(rojkov): drop io_uring dependency when it's fully integrated.
-           select({
-               "//bazel:linux": ["//source/common/io:io_uring_impl_lib"],
-               "//conditions:default": [],
-           }),
-)
-
 envoy_cc_library(
     name = "envoy_main_common_with_core_extensions_lib",
     srcs = [
```

Comment:
I'm slightly surprised to see stripped_main_base.h/.cc in both of these targets. I wonder if we should depend on stripped_main_base_lib here? WDYT?